pkg/core/state/contract.go

```diff
@@ -30,6 +30,12 @@ type ContractBase struct {
 	Manifest manifest.Manifest `json:"manifest"`
 }
 
+// Native holds information about native contract.
+type Native struct {
```

Comment:
`NativeContract`? `state.Native` is too broad.

Reply:
Fixed.